src/pages/developing/angular-tutorial/step-1.mdx

```diff
@@ -257,7 +257,7 @@ import { AppComponent } from './app.component';
 import { HeaderComponent } from './header/header.component';
 
 // carbon-components-angular default imports
-import { UIShellModule, IconModule } from 'carbon-components-angular';
+import { UIShellModule, IconModule, IconService } from 'carbon-components-angular';
```

Comment:
It's not clear to me why we need the `IconService` import here, since it's not used anywhere in the module ...

Reply:
it's been quite some time since I reported this, but basically it failed to compile without it.

Reply:
Hi @cal-smith, I just found why we need to add that. In this part of the tutorial:

> In the AppModule class we’ll add a constructor that provides us with an instance of IconService and call registerAll with an array of the icons we need to use.

and here is the code part of the tutorial:

```
constructor(protected iconService: IconService) {
  iconService.registerAll([
    Notification16,
    UserAvatar16,
    AppSwitcher16
  ]);
}
```

This may be an issue if you are using a text editor/IDE without an auto import plugin.

Reply:
Gotcha, didn't catch that ... It may be best if we introduce the import in the same place

Reply:
That way there's no surprises - if someone was following step by step they might skip adding the import before then since it's not obviously used